packages/vendure-plugin-payment-extensions/README.md

```diff
@@ -0,0 +1,33 @@
+# Vendure Order Payment Extensions
+
+## [Official documentation here](https://pinelab-plugins.com/plugin/vendure-plugin-payment-extensions)
+
+This `Vendure` plugin allows an `Order` to be settled without upfront payment.
+
+- `settleWithoutPaymentChecker`: Verifies if the logged-in customer belongs to a specified group, making them eligible for payment.
```

Comment:
I propose to rename this to `isCustomerInGroupPaymentChecker`, because it's independent of the settle without payment. That is what we use it for, but one could also use this for something like "If customer is in group CreditCard, then allow credit card payment method".